via/rewriter/templates/error.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
```

Comment:
This is copied from `pywb` only changing the name to "Via Error", but it's where we would add our own errors.